docs/3_troubleshooting.md

```diff
@@ -5,14 +5,16 @@ In case you encounter any issues while using ReVanced Manager, please refer to t
 - 💉 Patching fails with an error
 
   Make sure ReVanced Manager is up to date by following [🔄 Updating ReVanced Manager](2_3_updating.md) and select the **Default** button when choosing patches.
-  
+
 - 🚫 App not installed as package conflicts with an existing package
 
-  An existing installation of the app you're trying to patch is conflicting with the patched app. Uninstall the existing app before installing the patched app.
+  An existing installation of the app you're trying to patch is conflicting with the patched app (i.e., the existing version of the app doesn't allow upgrading to the patched version, or a different copy of the app causes a conflict). Uninstall the existing app before installing the patched app.
 
 - ❗️ Error code `135`, `139` or `1` when patching the app
 
-  Your device is not supported. Refer to the [Prerequisites](0_prerequisites.md) page for supported devices.
+  You may be trying to patch a split APK[^1], which is sometimes not supported by ReVanced. Sometimes apps are available in both split and non-split forms. If your existing installation of the app is a split APK, try obtaining a non-split version and see if that can be patched correctly.
```

Comment:
Extracting the essence of the text so that it is shorter:

```suggestion
  You may be trying to patch a split APK[^1]. This may not work under certain circumstances. In such a case, patch a full APK.
```